base/strings/util.jl

```diff
@@ -265,6 +269,7 @@ end
     chopsuffix(s::AbstractString, suffix::Union{AbstractString,Regex}) -> SubString
 
 Remove the suffix `suffix` from `s`. If `s` does not end with `suffix`, a string equal to `s` is returned.
+Note, for case-insensitive, e.g. for chopping off file endings like .EXE or .jl, use with a Regex.
```

Comment:
I don't understand what this sentence is trying to convey.

Reply:
You want to be able to chop off .jl, but not just, also .JL, (.Jl and .jL), and in general case-insensitive. I just providing one (common, most common) example. Did the .EXE part confuse? I was implying upper-case also a possibility (alse .exe, on Windows), not that you want to strip of .jl and .exe and any extension, you might someone, then with a different regex. How would you rephrase?